template-compose/app/src/main/java/co/nimblehq/template/compose/ui/theme/Theme.kt

```diff
@@ -1,4 +1,4 @@
-package co.nimblehq.template.compose.ui.screens.compose.theme
```

Comment:
@chornerman To get rid of  `@Suppress("FunctionNaming")`  Let's define `functionPattern` in `detekt-config` 🙏

Reply:
We have a separate task for this https://github.com/nimblehq/android-templates/issues/303 cc @Tuubz

Reply:
Hmm, But we already add it in `sample-compose` 🤔

Reply:
I think we better do re-check all Detekt config for all templates & samples in #303 task @Wadeewee @Tuubz